client/src/components/TaskCard/TaskCard.js

```diff
@@ -81,10 +80,9 @@ const TaskCard = ({
       {showReflection ? (
         <TaskReflectionModal
           readOnly={finished}
-          satisfaction={satisfaction}
-          reflectionBody={reflectionBody}
           onClose={toggleReflection}
           onDone={reflectionDoneHandler}
+          getTaskReflection={async () => getTaskReflection(id)}
```

Comment:
the getTaskReflection function should already be async, so we won't need to add in the `async` keyword here

Reply:
Removed `async` keyword.